docs/UserGuide.adoc

```diff
@@ -54,108 +53,226 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 === Viewing help : `help`
 
+Displays a help page +
 Format: `help`
 
-=== Adding a person: `add`
+// tag::createactivity[]
+=== Creating an activity: `create activity`
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Creates an activity to the activity list +
+Format: `create activity n/NAME d/DURATION [l/LOCATION] [dt/DATE_TIME]`
 
-[TIP]
-A person can have any number of tags (including 0)
+* DURATION is in hours. Shortest acceptable duration is 0.5 hours.
+* DATE_TIME is in dd/mm/yyyy HHHH format.
 
 Examples:
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `create activity n/Oscar d/21`
+* `create activity n/Ernest d/1 l/Tokyo dt/18/9/2019 0900`
 
-=== Listing all persons : `list`
+// end::createactivity[]
+=== Editing an activity: `edit activity`
 
-Shows a list of all persons in the address book. +
-Format: `list`
+Edits an activity +
+Format: `edit activity INDEX [n/NAME] [d/DURATION] [l/LOCATION]`
 
-=== Editing a person : `edit`
+* Edits the activity with the specified INDEX. The index refers to the index number shown in the displayed activity list. The index must be a positive integer 1, 2, 3, ...
+* At least one of the optional fields must be provided.
+* Existing values will be updated to the input values.
+
+Examples:
+
+* `edit activity 1 n/Dinner d/0.5 l/Hotel`
+
+Edits the name, duration and location of the 1st activity to be Dinner, 0.5 and Hotel respectively.
```

Comment:
Perhaps there could be better indentation to this line, to make it clear that it is the description of the given example.